src/components/SearchGridManualLoad.vue

```diff
@@ -223,4 +223,8 @@ export default {
       font-size: 1.2em;
     }
   }
+  .count{
+    padding-top: 10px;
+    padding-left: 20px;
```

Comment:
can you change this to `rem` units? You can use https://nekocalc.com/px-to-rem-converter to do the conversion

Reply:
I checked on every breakpoint I could and it’s properly spaced on all of them so I figured it was ok to keep as px. Go with what Breno says though @coderbabu10